src/Microsoft.AspNet.Mvc.Cors/CorsAuthorizationFilterFactory.cs

```diff
@@ -34,6 +34,9 @@ public int Order
             }
         }
 
+        /// <inheritdoc />
+        public bool ShouldCache => true;
```

Comment:
`public bool ShouldCache { get; } = true;` (to be consistent with everything else)